clothes-detection/src/App.js

```diff
@@ -1,12 +1,121 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './App.css';
+import MagicDropzone from 'react-magic-dropzone';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hi, I am react app.</h1>
-    </div>
-  );
+function getImageAttributes(imageUri) {
+  const yurieApiKey = 'AIzaSyCc2ZRO2cPmW50mHI8SDG1okNRzDo33gAU';
+  const majorApiKey = 'AIzaSyB_LRUShuGlrxDwNvR9FBsyyiMUUGXZTb0';
+  const url = `https://vision.googleapis.com/v1/images:annotate?alt=json&key=${yurieApiKey}`;
+  const requestParams = {
+    "requests": [
+      {
+        "image": {
+          "content" : imageUri,
+          "source": {
+            "imageUri": ''
+          }
+        },
+        "features": [
+          {
+            "maxResults": 0,
+            "type": "LABEL_DETECTION"
+          }
+        ]
+      }
+    ]
+  }
+  const options = {
+    method: 'POST',
+    body: JSON.stringify(requestParams),
+  };
+  fetch(url, options)
+  .then(response => response.json())
+  .then(imageAttributes => {
+    // figure out how to pass imageAttributes out to the react components
+    let labelArr = imageAttributes.responses[0].labelAnnotations;
+    console.log(labelArr);
```

Comment:
[**A comment** (PR comments with a purpose)](https://gist.github.com/echolabstech/a905ad02efba1ef6663c96ee4f251480):
Here is where we call the next function whose job it is to extract the labels. Think of the labels as a description for the clothes the user is looking for. We use that description to as the clothes data store for matches. Datastore is a general term for a database, a REST service that we query and get data from (like database as a service). [See class notes](https://gist.github.com/echolabstech/f5a7ce1d1d8a62c0cd505e94b9bac31c).